packages/core/src/theme-default/components/HomeHero/index.tsx

```diff
@@ -34,16 +37,16 @@ export function HomeHero({ frontmatter }: { frontmatter: FrontMatterMeta }) {
           </h1>
           {hero.text?.length && (
             <p
-              className={`mx-auto md:m-0 text-3xl sm:text-5xl md:text-6xl pb-2 font-bold z-10 max-w-392px ${
```

Comment:
@sanyuan0704 It seems that `max-w-392px` is not a valid Tailwind CSS class name, should we delete it?

Reply:
yeah

Reply:
done